docs/source/basic/install.rst

```diff
@@ -3,44 +3,61 @@
 Installation
 ============
 
-* Clone alpaka from github.com
-
 .. code-block::
 
+  # Clone alpaka from github.com
   git clone https://github.com/alpaka-group/alpaka
   cd alpaka
-
-* Install alpaka
-
-.. code-block::
-
-  # git clone https://github.com/alpaka-group/alpaka
-  # cd alpaka
   mkdir build && cd build
   cmake -DCMAKE_INSTALL_PREFIX=/install/ ..
   cmake --install .
 
-* Configure Accelerators
+Tests and Examples
+++++++++++++++++++
 
-.. code-block::
-
-  # ..
-  cmake -DALPAKA_ACC_GPU_CUDA_ENABLE=ON ..
-
-* Build an example
+**Build and run examples:**
 
 .. code-block::
 
   # ..
   cmake -Dalpaka_BUILD_EXAMPLES=ON ..
-  make vectorAdd
+  cmake --build . -t vectorAdd
   ./example/vectorAdd/vectorAdd # execution
 
-* Build and run tests
+**Build and run tests:**
 
 .. code-block::
 
   # ..
   cmake -DBUILD_TESTING=ON ..
-  make
+  cmake --build .
   ctest
+
+**Enable accelerators:**
+
+``alpaka`` uses different accelerators to execute kernels on different processors. To use a specific accelerator in ``alpaka``, two steps are required.
+
+1. Enable the accelerator during the ``CMake`` configuration time of the project.
+2. Select a specific accelerator in the source code.
+
+By default, no accelerator is enabled because some combination of compilers and accelerator does not work, see the table of `supported compiler <https://github.com/alpaka-group/alpaka#supported-compilers>`_. To enable an accelerator, you must set a cmake flag via ``cmake .. -DALPAKA_ACC_<acc>_ENABLE=ON`` when you create a new build. The following example shows how to enable the CUDA accelerator and build an ``alpaka`` project:
```

Comment:
```suggestion
By default, no accelerator is enabled because some combinations of compilers and accelerators do not work, see the table of `supported compilers <https://github.com/alpaka-group/alpaka#supported-compilers>`_. To enable an accelerator, you must set a CMake flag via ``cmake .. -DALPAKA_ACC_<acc>_ENABLE=ON`` when you create a new build. The following example shows how to enable the CUDA accelerator and build an ``alpaka`` project:
```